core/tests/puppeteer/lighthouse_setup.js

```diff
@@ -268,6 +269,22 @@ const getSkillEditorUrl = async function(browser, page) {
   }
 };
 
+const generateDataForTopicAndStoryPlayer = async function(browser, page) {
+  try {
+    // eslint-disable-next-line dot-notation
+    await page.goto('http://127.0.0.1:8181/admin#/activities', { waitUntil: networkIdle });
+
+    await page.waitForSelector(generateTopicButton);
+    await page.click(generateTopicButton);
+
+    await page.waitForTimeout(20000);
+  } catch (e) {
+    // eslint-disable-next-line no-console
+    console.log(e);
+    process.exit(1);
```

Comment:
Why is catching the error, printing it, and then quitting better than just letting the error propagate normally?

Reply:
@U8NWXD Please look the above functions:  getSkillEditorUrl, getStoryEditorUrl, getTopicEditorUrl. They all exit if the error is occured, also when there is an error it means that the story player and topic player urls won't be available to run the lighthouse on them, so it will give error then. Do you still think I should remove the exit?